samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -53,8 +52,8 @@ BUILD_ASSERT(sizeof(CONFIG_APP_LWM2M_SERVER) > 1,
 #define LWM2M_SECURITY_CERTIFICATE 2
 #define LWM2M_SECURITY_NO_SEC 3
 
-static uint8_t endpoint_name[ENDPOINT_NAME_LEN + 1];
-static uint8_t imei_buf[IMEI_LEN + 3]; /* account for /n/r */
+static uint8_t endpoint_name[ENDPOINT_NAME_LEN + 1 + 4];
```

Comment:
Why do we need to increase the size of `endpoint_name`?

Reply:
Leftover from a previous fix related to `snprintf` truncation since destination buffer was too small.